app/controllers/App.scala

```diff
@@ -41,7 +41,15 @@ object App extends Controller with PanDomainAuthActions {
         }
       }
 
-      val clientConfig = ClientConfig(Config().capiUrl, Config().capiKey, allTags, permittedTags.toList, permissions, "/reauth")
+      val clientConfig = ClientConfig(
+        capiUrl = Config().capiUrl,
+        capiPreviewUrl = "/support/previewCapi",
+        capiKey = Config().capiKey,
```

Comment:
This is because we are only using preview for the batch tagger, and for live CAPI we are going direct from client (To fetch tag usage stats etc)